src/components/index.tsx

```diff
@@ -2,7 +2,16 @@ import React, { useRef, useState } from "react"
 import Editor from "@monaco-editor/react"
 import useScript from "../hooks/useScript"
 
-const Quiz: React.FunctionComponent = props => {
+type quizProps = {
```

Comment:
The convention is to use UpperCamelCase in type names. (https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#type)

Reply:
Thanks, fixed.